Aulas-HTML/exercicios/aula2/index.html

```diff
@@ -1,23 +1,61 @@
-<!--
-Exercicio 2:
-Vamos tentar reproduzir a estrutura de um layout? 
-Escolha um site da sua preferência e tente identificar 
-nele os elementos que vimos hoje (<header>, <footer>,
-<nav>, <article>). Depois, crie um arquivo HTML e use
-as tags de uma forma que você acha que o conteúdo do site
-que você escolheu fique organizado.
+<!DOCTYPE html>
+<html lang="pt">
+  <head>
+    <title>Exercicio 2 - Thais Dias</title>
+    <meta charset="utf-8">
+    <meta name="description" content="Exercício 2">
+    <meta name="keywords" content="HTML5">
+    <meta name="author" content="Thais Wemberlaine">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
 
-Coloque o link do site que você se baseou na descriçaão do PR 
-
-Não se preocupe em copiar os conteúdos ou em deixá-los bonitos.
-Lembre-se: HTML é sobre organizar o nosso conteúdo de uma forma semântica. 
-
-Entrega:
- - Fazer fork do repositório pro seu usuário do github (caso já tenha feito o fork desse repositório, 
-   não precisa fazer novamente. Faça a atualização do fork pra ter os novos arquivos da aula)
- - Fazer o clone do repositório "forkado" da sua conta do github para a sua máquina
- - Colocar a resolução do exercício nesse arquivo (aulas-html/exercicios/aula2/index.html)
- - Dar git push das alterações
- - Abrir um pull request para o repositório original (da Womakers)
- - Acompanhar o prazo e marcar a entrega no Classroom
--->
+  <body>
+    <header>
+      <img src="images/amazon-logo.jpg" alt="Logo da amazon.com.br"/>
+      <nav>
+        <h1>Sua Amazon.com.br</h1>
+        <p>Ofertas do Dia</p>
+        <p>Mais vendidos</p>
+        <p>Livros</p>
+        <p>Atendimento ao cliente</p>
+        <p>Prime Video|Assista a filmes e séries</p>
+      </nav>
+      <nav>
+        Dispositivos Amazon
+        <p>Echo e Alexa</p>
+        <p>Kindle</p>
+        <p>Fire TV</p>
+        <p>Acessórios</p>
+        <p>Casa inteligente</p>
+        <p>Conheça a Alexa</p>
+        <p>Amazon Music</p>
+      </nav>
+    </header>
```

Comment:
Se tudo que estiver aqui dentro desta tag `<header></header>` for um só menu, só precisa utilizar o `<nav></nav>` apenas 1 vez